effekt/jvm/src/test/scala/effekt/core/RenamerTests.scala

```diff
@@ -9,9 +11,9 @@ class RenamerTests extends CoreTests {
                       names: Names = Names(defaultNames))(using munit.Location) = {
     val pInput = parse(input, "input", names)
     val pExpected = parse(renamed, "expected", names)
-    val renamer = new Renamer(names, "renamed") // use "renamed" as prefix so we can refer to it
+    val renamer = new Renamer(names, "renamed")
     val obtained = renamer(pInput)
-    shouldBeEqual(obtained, pExpected, clue)
+    assertAlphaEquivalent(obtained, pExpected, clue)
   }
```

Comment:
Given that we check for alpha-equivalence anyway, we could drop one argument and not have expected, couldn't we? 🤔

(i.e. just have the input in each case)

Reply:
Applied, thanks!